api/Dockerfile

```diff
@@ -1,12 +1,24 @@
-FROM ubuntu:latest
-ENV SERVER_HOST=0.0.0.0
-RUN apt-get update && apt-get install -y \
-  python-pip \
-  python-dev \
-  build-essential
-COPY . /app
-WORKDIR /app
-RUN pip install --upgrade pip
-RUN pip install -r requirements.txt
-ENTRYPOINT ["python"]
-CMD ["views.py"]
+FROM httpd:2.4.23
+ENV http_proxy=http://proxy.autozone.com:8080/ SERVER_HOST=0.0.0.0
+WORKDIR /opt
+RUN apt-get update && apt-get install --no-install-recommends -y unzip curl python-pip python-dev build-essential && \
+      curl -k https://infosec.autozone.com/files/openam/Apache_v24_Linux_64bit_4.1.0.zip > /var/tmp/agent.zip && \
+      unzip /var/tmp/agent.zip && rm /var/tmp/agent.zip && \
+      echo "{OpenAM Password}" > /var/tmp/pw.txt  && chmod 0400 /var/tmp/pw.txt && \
```

Comment:
Rather than making this part of building the image, make this happen at runtime. That means your pw.txt should be exposed in the dockerfile as a volume, then part of the deployment of this image is to map the pw.txt file from the host into the container and execute that agentadmin command in it (`docker exec -it <container name> /bin/sh` to get into shell of a running container).